scripts/aeron/live-recording-client

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+##
+## Copyright 2015-2020 Real Logic Limited.
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+## https://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##
+
+AERON_DIR="`pwd`"
+
+cd ..
+./run-java -Daeron.benchmarks.rtt.messageTransceiver=uk.co.real_logic.benchmarks.rtt.aeron.LiveRecordingMessageTransceiver \
+  uk.co.real_logic.benchmarks.rtt.LoadTestRig \
+  ${AERON_DIR}/low-latency-driver.properties \
+  ${AERON_DIR}/low-latency-archive.properties \
+  ${AERON_DIR}/benchmark.properties
```

Comment:
Could pick up further properties files on the command line and append. This would allow for overriding as last one in winds.